src/test/resources/spec/concordion/common/command/example/FixtureWithoutExampleInitialisation.html

```diff
@@ -2,7 +2,7 @@
 <link href="../../../../concordion.css" rel="stylesheet" type="text/css" />
 <body>
 
-<h1>Field Initialisation in Fixtures</h1>
+<h1 c:execute="resetFieldInitialisationCounter()">Field Initialisation in Fixtures</h1>
```

Comment:
Just checking why we need to reset this counter? It could potentially make this test invalid, with the test checking that the fields are only initialised once.
The issue seems to be that the fixture is created twice when running JUnit5, once from JUnitPlatformLauncher and once from DefaultConcordionRunner, with both calling ConcordionRunner which creates the fixture in its constructor on line 51.
Running the same test with the current production version, we only create this fixture once from the JUnitTestClassExecutor.
I still need to do a bit more digging to find out why (I'm a bit rusty on this code and don't fully understand the JUnit5 model yet!).
Have you got any thoughts on this?

Reply:
Thanks @nigelcharman. You're right. The `ConcordionRunner` is instantiating the fixture object too early. Please see https://github.com/concordion/concordion/pull/243#issuecomment-1322873077. Will it be okay to submit a _separate_ PR to modify `ConcordionRunner` to work on _both_ JUnit 4 _and_ JUnit Vintage? Then, _this_ PR to create/add a JUnit Platform engine (`ConcordionTestEngine`).